tests/test_install.sh

```diff
@@ -1,192 +1,284 @@
-  echo "#  Do you wish to install the OWASP Juice Shop Application?"
+  echo "#  Install OWASP Juice Shop Application"
+  echo "#  "
+  echo "#  Documentation: https://artifacthub.io/packages/helm/securecodebox/juice-shop"
   echo "########################################################################################"
   read -r -p 'Install OWASP Juice Shop [y/n]: '
 
   if [[ ${REPLY} == 'y' ]]; then
     helm repo add securecodebox https://charts.securecodebox.io/
-    helm install juice-shop securecodebox/juice-shop --namespace juice-shop --create-namespace \
+
+    echo "########################################################################################"
+    echo "#  Waiting for Juice Shop to start"
+    echo "########################################################################################"      
+
+    helm install juice-shop-${RELEASE_VER} securecodebox/juice-shop --namespace juice-shop --create-namespace \
       --set ingress.enabled=true \
       --set "ingress.hosts[0].host=localhost,ingress.hosts[0].paths[0].path=/" \
       --set "ingress.tls[0].hosts[0]=localhost,ingress.tls[0].secretName=" \
-      --set ingress.pathType=Prefix
+      --set ingress.pathType=Prefix \
+      --wait --timeout=60s
 
+  else
     echo "########################################################################################"
-    echo "#  Waiting for Juice Shop to start"
+    echo "#  Skipping Juice Shop installation"
     echo "########################################################################################"
+  fi
+
+  echo "########################################################################################"
+  echo "#  Do you wish to install Redis?"
+  echo "#  "
+  echo "#   This is required by Codesealer.  Either use this installation helm chart"
+  echo "#   or follow the Codesealer documentation to install your own version"
+  echo "#  "
+  echo "#   -- Using github.com/bitnami/charts repository"
+  echo "#      Documentation: https://github.com/bitnami/charts/blob/main/bitnami/redis/README.md"
+  echo "########################################################################################"
+  read -r -p 'Install Redis in single master mode [y/n]: '
+  if [[ ${REPLY} == 'y' ]]; then
 
-    kubectl wait --namespace juice-shop \
-    --for=condition=ready pod \
-    --selector=app.kubernetes.io/name=juice-shop \
-    --timeout=90s
+    echo "########################################################################################"
+    echo "#  Waiting for Redis to start"
+    echo "########################################################################################"      
+    helm install redis-${RELEASE_VER} oci://registry-1.docker.io/bitnamicharts/redis \
+    --namespace ${REDIS_NAMESPACE} --create-namespace \
+    --set auth.enabled=true \
+    --set replica.replicaCount=1 \
+    --wait --timeout=60s
+
+    # --set sentinel.enabled=false \
+    # --set auth.sentinel=false \
```

Comment:
Please don't leave random commented lines lying around.

Reply:
I originally intended this script to be a reference for Adam to create the copy paste commands.  It was to illustrate how the parameters could be updated by him.  Do we plan on providing a script to the user or will this be for us to test new releases?  This was here to illustrate if we wanted to turn on Sentinel and test it.

Reply:
Well, this is a public repository, so yes, we're providing this to our users. That's also why I'm trying to keep it fairly polished. 

If we want to present the possible options to the users, we should do that in the right place, i.e. the readme, either explicitly or by pointing the user to `helm show values`.